clang/test/AST/HLSL/RWBuffer-AST.hlsl

```diff
@@ -29,15 +29,21 @@ RWBuffer<float> Buffer;
 // CHECK-NEXT: CXXRecordDecl 0x{{[0-9A-Fa-f]+}} <<invalid sloc>> <invalid sloc> implicit class RWBuffer definition
 
 // CHECK: FinalAttr 0x{{[0-9A-Fa-f]+}} <<invalid sloc>> Implicit final
-// CHECK-NEXT: implicit h 'element_type * {{\[\[}}hlsl::resource_class(UAV)]] {{\[\[}}hlsl::contained_type(element_type)]]':'element_type *'
+// CHECK-NEXT: implicit h 'element_type *
```

Comment:
(prexisting...but...)

Aren't these regexes?  so `element_type *` matches element_type followed by zero or more spaces? So the string `element_typefoo` would match.

Reply:
This is not a regex - it would have to be enclosed in `{{}}`.